Python/initconfig.c

```diff
@@ -317,6 +318,7 @@ The following implementation-specific options are available:\n\
 -X perf: support the Linux \"perf\" profiler; also PYTHONPERFSUPPORT=1\n\
 -X perf_jit: support the Linux \"perf\" profiler with DWARF support;\n\
          also PYTHON_PERF_JIT_SUPPORT=1\n\
+-X disable-remote-debug: disable remote debugging; also PYTHON_DISABLE_REMOTE_DEBUG\n\
```

Comment:
All of the others use `_` instead of `-` so this should probably be
```suggestion
-X disable_remote_debug: disable remote debugging; also PYTHON_DISABLE_REMOTE_DEBUG\n\
```